extern/boostd-data/yugabyte/service.go

```diff
@@ -39,6 +38,8 @@ type DBSettings struct {
 	PayloadPiecesParallelism int
 	// CQL timeout in seconds
 	CQLTimeout int
+	// Number of records per insert batch
+	InsertBatchSize int
```

Comment:
I find this confusing... we have `InsertBatchSize` within Settings, but later it is used outside of settings, directly in `Store` - is it possible to be consistent somehow and have it at one place?